website/docs/reference/widgets/select.md

```diff
@@ -127,15 +127,43 @@ Style properties allow you to change the look and feel of the widget.
 | **Box Shadow**       | Allows you to choose from the available shadow styles.   |
 
-### Server-side Filtering
+### Server-side filtering
 
-The Select widget has the option to configure server-side filtering, where search queries are sent to the backend, and responses are used to populate options on the Select widget. When enabling server-side filtering in the widget, please update the default value to contain both `label` and `value` in this format `{"label":<label>, "value": <value>}` if the default value is not present in the default options.
-
-The video below explains how to configure this.
+The Select widget has the option to configure server-side filtering, where search queries are sent to the back-end, and responses are used to populate options on the Select widget. When enabling server-side filtering in the widget, please update the default value to contain both `label` and `value` in this format `{"label":<label>, "value": <value>}` if the default value isn't present in the default options.
 
 <VideoEmbed host="youtube" videoId="QDmTwRaLzHg" title="Server-side Filtering" caption="Server-side Filtering"/>
 
+As shown in the video, follow these steps to configure server side filtering:
+
+1. Drag a Select widget onto the canvas.
+1. Create a [query](/core-concepts/data-access-and-binding/querying-a-database/) `get_users` that retrieves user data to use in the widget. For example, try working with Appsmith's `users` Postgresql test database. Use the following statement as the query:
+  ```sql
+  SELECT id, name FROM users ORDER BY id LIMIT 10;
+  ```
+1. In the Select widget's properties pane, add the query's `data` to the widget's **Options** property. To control how the returned data is structured, use the JS [`map()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) method:
+  ```javascript
+  {{
+    get_users.data.map( user => {
+      return { label: user.name, value: user.id }
+    })
+  }}
+  ```
+1. Enable the Select widget's **Server Side Filtering** property, and set the **onFilterUpdate** event to execute your query.
+  ```javascript
+  {{ get_users.run() }}
+  ```
+  ![](/img/as_select_filtering.png)
+1. Now that the widget is configured for server side filtering, update the `get_users` query with a `WHERE` statement that incorporates the filter text from the Select widget:
+  ```sql
+  SELECT id, name FROM users
+  WHERE name LIKE '{{'%'+ Select2.filterText + '%'}}'
+  ORDER BY id LIMIT 10;
+  ```
+
+Now when you open your Select widget, you can enter text to narrow the query's results.
+
+You can also take a look at what this behavior looks like by visiting the sample app for this topic ([Select Widget - Server Side Filtering](https://app.appsmith.com/applications/61fbdf232cd3d95ca414b805/pages/6215d4742882606a1df5c695)). To begin with, there are only 10 results in the widget; if you enter text, the list updates itself with new entries that match your filter text.
```

Comment:
Remove pronouns

View the sample app for ([Select Widget - Server Side Filtering](https://app.appsmith.com/applications/61fbdf232cd3d95ca414b805/pages/6215d4742882606a1df5c695)) 

I think 'To begin with, there are only 10 results in the widget; if you enter text, the list updates itself with new entries that match your filter text.' should be removed from the docs. 

@pranavkanade can you add that description to the sample app please? Thanks